pkg/util/file_extension.go

```diff
@@ -0,0 +1,9 @@
+package util
+
+func GetFileExtensionByContentType(contentType string) string {
```

Comment:
I don't think this function should be in pkg/util, but instead as a private function inside the package where it's used. There's no reason to make pkg/util even fatter than it is already. In general it's an anti-pattern to have a util package.

Reply:
Done